### PR TITLE
[TOOLS-2124] touch time by week effort in hours

### PR DESCRIPTION
**Pull request by marcokuchla**
* Change Touch time by week effort's unit from seconds to hours;

### CODE REVIEW

File: kpis/src/main/java/objective/taskboard/followup/kpi/touchTime/TouchTimeByWeekDataProvider.java [hunk 1]
```diff
@@ -101,9 +102,9 @@ private TouchTimeChartByWeekDataPoint getPointForDateAndStatus(Range<LocalDate>
             List<IssueKpi> issuesToCount = issues.stream().filter(filter).collect(Collectors.toList());
             ZonedDateTime lastDayOfWeek = endOfWeek(week);
             double averageEffort = issuesToCount.stream().mapToLong(issue -> issue.getEffortUntilDate(status, lastDayOfWeek)).average().orElse(0d);
-            
+            double averageEffortInHours = DateTimeUtils.secondsToHours(averageEffort);
```
Comment (JuniorMiqueletti) [inline kpis/src/main/java/objective/taskboard/followup/kpi/touchTime/TouchTimeByWeekDataProvider.java]: Minor. I suggest to you put this "transformation" on lambda above and continue using the "avarageEffort"...
like:

            double averageEffort = 
                issuesToCount.stream()
                    .mapToLong(issue -> issue.getEffortUntilDate(status, lastDayOfWeek))
                    .mapToDouble(DateTimeUtils::secondsToHours)
                    .average().orElse(0D);

**Comment by JuniorMiqueletti** [comment]
@marcokuchla, now the Taskboard doesn't have data to see this chart on the "TestMain"...
if it is easy, that would be interesting :)

File: core/src/main/java/objective/taskboard/utils/DateTimeUtils.java [hunk 2]
```diff
@@ -186,4 +186,8 @@ public T next() {
     public static double secondsToHours(double seconds) {
         return seconds / DateTimeUtils.HOUR_IN_SECONDS;
     }
+
+    public static long hoursToSeconds(double hours) {
```
[the remark this answers is not in the document]
Comment (marcokuchla) [reply core/src/main/java/objective/taskboard/utils/DateTimeUtils.java]: Done.

File: kpis/src/main/java/objective/taskboard/followup/kpi/touchTime/TouchTimeByWeekDataProvider.java (hunk 1 above)
Comment (marcokuchla) [reply kpis/src/main/java/objective/taskboard/followup/kpi/touchTime/TouchTimeByWeekDataProvider.java]: Fixed.

**Comment by marcokuchla** [comment]
> @marcokuchla, now the Taskboard doesn't have data to see this chart on the "TestMain"...
> if it is easy, that would be interesting :)

Fixed partially:
The charts now display "Subtask" level's data, but the Touch Time charts don't have data because the TestMain doesn't import worklogs necessary for displaying the data.

**Pull request merged**